pkg/connector/matrix.go

```diff
@@ -166,6 +167,23 @@ func (l *LinkedInClient) HandleMatrixMessage(ctx context.Context, msg *bridgev2.
 		}
 	}
 
+	if msg.ReplyTo != nil {
+		intent := l.userLogin.Bridge.Matrix.GhostIntent(l.userID)
+		evt, err := intent.GetEvent(ctx, msg.Event.RoomID, msg.ReplyTo.MXID)
+		if err != nil {
+			return nil, err
```

Comment:
This should probably not be a fatal error, more like a log. What do replies look like if you don't provide the text? Does it still show some indication it's a reply?

Reply:
Should also handle both evt and err being nil as per the sdk PR

The Matrix connector won't return nil without an error, but no need to require the matrix interface to always return an error